sale_product_approval/models/sale_order.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from odoo import SUPERUSER_ID, _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    exceptions_sale_approval_confirm = fields.Boolean(
+        compute="_compute_exceptions", string="Exception", default=False
```

Comment:
Shouldn't this renamed to `_compute_exceptions_sale_approval_confirm` to avoid naming conflicts ?